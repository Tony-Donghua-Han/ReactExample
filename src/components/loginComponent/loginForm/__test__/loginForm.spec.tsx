import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import LoginForm from "../loginForm.component";

describe("LoginForm", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<LoginForm onSubmit={(): void => {}} />);
    });

    test("should render without crushing", (): void => {
        expect(wrapper.dive().debug()).toMatchSnapshot();
    });
});
