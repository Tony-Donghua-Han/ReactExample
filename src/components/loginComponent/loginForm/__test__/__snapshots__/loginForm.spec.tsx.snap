// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should render without crushing 1`] = `
"<ContextProvider value={{...}}>
  <Form className=\\"loginForm\\">
    <FormikConnect(FieldInner) type=\\"email\\" name=\\"email\\" className=\\"inputBox\\" placeholder=\\"Email\\" />
    <FormikConnect(ErrorMessageImpl) name=\\"email\\" className=\\"alert\\" component=\\"div\\" />
    <FormikConnect(FieldInner) type=\\"password\\" name=\\"password\\" className=\\"inputBox\\" placeholder=\\"Password\\" />
    <FormikConnect(ErrorMessageImpl) name=\\"password\\" className=\\"alert\\" component=\\"div\\" />
    <button type=\\"submit\\" disabled={false} className=\\"submitButton\\">
      Submit
    </button>
  </Form>
</ContextProvider>"
`;
